curvefs/src/client/inode_wrapper.h

```diff
@@ -363,6 +377,21 @@ class InodeWrapper : public std::enable_shared_from_this<InodeWrapper> {
 
     CURVEFS_ERROR RefreshVolumeExtent();
 
+    bool NeedRefreshData() {
+        uint64_t size = GetS3ChunkSize();
+        if (size >= maxDataSize_ &&
+            ::curve::common::TimeUtility::GetTimeofDaySec()
+            - lastRefreshTime_ > refreshDataInterval_) {
+            VLOG(6) << "EliminateLargeS3ChunkInfo size = " << size
+                    << ", max = " << maxDataSize_
+                    << ", inodeId = " << inode_.inodeid();
+            inode_.clear_s3chunkinfomap();
```

Comment:
Why is the infomap of the inode emptied in the judgment function?RefreshS3ChunkInfo will handle old chunkinfomap.

Reply:
fixed